src/Field.php

```diff
@@ -450,7 +450,7 @@ public function getQueryArguments($operator, $value): array
      */
     public function isEditable(): bool
     {
-        return $this->ui['editable'] ?? !$this->read_only && !$this->never_persist && !$this->system;
+        return $this->ui['editable'] ?? !$this->read_only && !$this->never_persist && !$this->never_save && !$this->system;
```

Comment:
what is the real different between `read_only` and `never_save`?

Reply:
@DarkSide666 @abbadon1334 @georgehristov any idea?

Reply:
I tried the condition above and it does not fix the atk4/filestore issue described in https://github.com/atk4/data/issues/929